saleor/wombat/__init__.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+import requests
+import logging
+
+logger = logging.getLogger('wombat')
+
+
+class WombatClient(object):
+    endpoint = 'https://push.wombat.co'
```

Comment:
Actually wombat doesn't have sandbox mode. There is only limited trial account, so endpoint is always the same.